src/index.html

```diff
@@ -12,6 +12,124 @@
+      <fieldset class="groups">
+        <legend>Personal information</legend>
+        <label class="input">Surname:
+          <input 
+            type="text" 
+            name="surname" 
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="input">Name:
+          <input 
+            type="text" 
+            name="name" 
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label class="input">How old are You?
+          <input 
+            type="number" 
+            name="ages"
+            required
+            value="12"
+            min="1"
+            max="100"
+          >
+        </label>
+        <label class="input">Full date of birth:
+          <input type="date" name="date">
+        </label>
+        <label>I accept the term of the agreement
+          <input 
+            type="checkbox" 
+            name="termsOfTheAgreement" 
+            required
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>Registration</legend>
+        <label class="input">E-mail:
+          <input 
+            type="email" 
+            name="email" 
+            placeholder="email@example.com"
+            required
+            autocomplete="off"
+          >
+        </label>
+        <label>Password:
+          <input 
+            type="password" 
+            name="password" 
+            required
+            minlength="5"
+            maxlength="15"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>An interesting fact about you!</legend>
+        <label class="input">Do you love cats?
+          <label>
+            <input type="radio" name="loveCats">Yes
+          </label>
+          <label>
+            <input type="radio" name="loveCats">No
+          </label>
+        </label>
+        <label class="input">What is your favorite color?
+          <input type="color" name="favColor">
+        </label>
+        <label class="input">What time do you go to bed?
+          <input type="time" name="time">
+        </label>
+        <label class="input">What are your favorite brand of cars?
+          <select name="brandOfCars" multiple>
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+        <label>How do you rate your work?
+          <input 
+            type="range" 
+            name="rateWork" 
+            min="0" 
+            max="100" 
+            value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="groups">
+        <legend>Additional info:</legend>
+        <label class="input">Comments:
+          <textarea 
+            name="comments" 
+            cols="30" 
+            rows="2"
+            autocomplete="off"
+          >
+          </textarea>
```

Comment:
```suggestion
          >
          </textarea>
```

https://mate.academy/learn/css-advanced/forms#/video/885